<?xml version="1.0" encoding="UTF-8"?>
<testng-results skipped="0" failed="3" ignored="0" total="3" passed="0">
  <reporter-output>
  </reporter-output>
  <suite name="Ecommerce test run" duration-ms="3790" started-at="2017-12-05T15:26:34Z" finished-at="2017-12-05T15:26:38Z">
    <groups>
    </groups>
    <test name="Place Order" duration-ms="3790" started-at="2017-12-05T15:26:34Z" finished-at="2017-12-05T15:26:38Z">
      <class name="com.Ecomm.framework.OrderFlow_Mobile">
        <test-method status="FAIL" signature="Addtocart_mob(java.lang.String, java.lang.String, java.lang.String, java.lang.String)[pri:1, instance:com.Ecomm.framework.OrderFlow_Mobile@29ca901e]" name="Addtocart_mob" duration-ms="2560" started-at="2017-12-05T20:56:34Z" finished-at="2017-12-05T20:56:36Z">
          <params>
            <param index="0">
              <value>
                <![CDATA[iOS]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[11.1.2]]>
              </value>
            </param>
            <param index="2">
              <value>
                <![CDATA[iPhone-6]]>
              </value>
            </param>
            <param index="3">
              <value>
                <![CDATA[Georgia]]>
              </value>
            </param>
          </params>
          <exception class="com.perfecto.reportium.exception.ReportiumException">
            <message>
              <![CDATA[Missing required webdriver argument. Call your builder's withWebDriver() method]]>
            </message>
            <full-stacktrace>
              <![CDATA[com.perfecto.reportium.exception.ReportiumException: Missing required webdriver argument. Call your builder's withWebDriver() method
	at com.perfecto.reportium.model.PerfectoExecutionContext.<init>(PerfectoExecutionContext.java:23)
	at com.perfecto.reportium.model.PerfectoExecutionContext$PerfectoExecutionContextBuilder.build(PerfectoExecutionContext.java:42)
	at com.Ecomm.framework.OrderFlow_Mobile.Addtocart_mob(OrderFlow_Mobile.java:52)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:108)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:669)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:877)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1201)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
	at org.testng.TestRunner.privateRun(TestRunner.java:776)
	at org.testng.TestRunner.run(TestRunner.java:634)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:425)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:420)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:385)
	at org.testng.SuiteRunner.run(SuiteRunner.java:334)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1318)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1243)
	at org.testng.TestNG.runSuites(TestNG.java:1161)
	at org.testng.TestNG.run(TestNG.java:1129)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:114)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- com.perfecto.reportium.exception.ReportiumException -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- Addtocart_mob -->
        <test-method status="FAIL" signature="Checkoutcart_mob(java.lang.String, java.lang.String, java.lang.String, java.lang.String)[pri:2, instance:com.Ecomm.framework.OrderFlow_Mobile@29ca901e]" name="Checkoutcart_mob" duration-ms="620" started-at="2017-12-05T20:56:36Z" finished-at="2017-12-05T20:56:37Z">
          <params>
            <param index="0">
              <value>
                <![CDATA[iOS]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[11.1.2]]>
              </value>
            </param>
            <param index="2">
              <value>
                <![CDATA[iPhone-6]]>
              </value>
            </param>
            <param index="3">
              <value>
                <![CDATA[Georgia]]>
              </value>
            </param>
          </params>
          <exception class="com.perfecto.reportium.exception.ReportiumException">
            <message>
              <![CDATA[Missing required webdriver argument. Call your builder's withWebDriver() method]]>
            </message>
            <full-stacktrace>
              <![CDATA[com.perfecto.reportium.exception.ReportiumException: Missing required webdriver argument. Call your builder's withWebDriver() method
	at com.perfecto.reportium.model.PerfectoExecutionContext.<init>(PerfectoExecutionContext.java:23)
	at com.perfecto.reportium.model.PerfectoExecutionContext$PerfectoExecutionContextBuilder.build(PerfectoExecutionContext.java:42)
	at com.Ecomm.framework.OrderFlow_Mobile.Checkoutcart_mob(OrderFlow_Mobile.java:116)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:108)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:669)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:877)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1201)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
	at org.testng.TestRunner.privateRun(TestRunner.java:776)
	at org.testng.TestRunner.run(TestRunner.java:634)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:425)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:420)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:385)
	at org.testng.SuiteRunner.run(SuiteRunner.java:334)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1318)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1243)
	at org.testng.TestNG.runSuites(TestNG.java:1161)
	at org.testng.TestNG.run(TestNG.java:1129)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:114)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- com.perfecto.reportium.exception.ReportiumException -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- Checkoutcart_mob -->
        <test-method status="FAIL" signature="PlaceOrder_mob(java.lang.String, java.lang.String, java.lang.String, java.lang.String)[pri:3, instance:com.Ecomm.framework.OrderFlow_Mobile@29ca901e]" name="PlaceOrder_mob" duration-ms="560" started-at="2017-12-05T20:56:37Z" finished-at="2017-12-05T20:56:38Z">
          <params>
            <param index="0">
              <value>
                <![CDATA[iOS]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[11.1.2]]>
              </value>
            </param>
            <param index="2">
              <value>
                <![CDATA[iPhone-6]]>
              </value>
            </param>
            <param index="3">
              <value>
                <![CDATA[Georgia]]>
              </value>
            </param>
          </params>
          <exception class="com.perfecto.reportium.exception.ReportiumException">
            <message>
              <![CDATA[Missing required webdriver argument. Call your builder's withWebDriver() method]]>
            </message>
            <full-stacktrace>
              <![CDATA[com.perfecto.reportium.exception.ReportiumException: Missing required webdriver argument. Call your builder's withWebDriver() method
	at com.perfecto.reportium.model.PerfectoExecutionContext.<init>(PerfectoExecutionContext.java:23)
	at com.perfecto.reportium.model.PerfectoExecutionContext$PerfectoExecutionContextBuilder.build(PerfectoExecutionContext.java:42)
	at com.Ecomm.framework.OrderFlow_Mobile.PlaceOrder_mob(OrderFlow_Mobile.java:190)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:108)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:669)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:877)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1201)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
	at org.testng.TestRunner.privateRun(TestRunner.java:776)
	at org.testng.TestRunner.run(TestRunner.java:634)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:425)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:420)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:385)
	at org.testng.SuiteRunner.run(SuiteRunner.java:334)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1318)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1243)
	at org.testng.TestNG.runSuites(TestNG.java:1161)
	at org.testng.TestNG.run(TestNG.java:1129)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:114)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- com.perfecto.reportium.exception.ReportiumException -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- PlaceOrder_mob -->
      </class> <!-- com.Ecomm.framework.OrderFlow_Mobile -->
    </test> <!-- Place Order -->
  </suite> <!-- Ecommerce test run -->
</testng-results>
